charts/consul/templates/gateway-resources-clusterrole.yaml

```diff
@@ -10,6 +10,15 @@ metadata:
     release: {{ .Release.Name }}
     component: gateway-resources
 rules:
+  - apiGroups:
+    - consul.hashicorp.com
```

Comment:
MeshGateway doesn't actually exist in this API group right?

Reply:
ah no it does not, that's a bad copy pasta